elections.md

```diff
@@ -101,4 +91,16 @@ treated as “brand free” activities.
 ## Emeritus Term
 
 Members of the steering committee will graduate to becoming Emeritus members of
-the steering committee.
\ No newline at end of file
+the steering committee.
+
+## Steering committee and election officer recusal
+
+Steering committee members and the appointed election offers pledge to recuse
+themselves from any form of electioneering, including campaigning, nominating,
+endorsing, or even asking people to run.
+
+[kubernetes/community/events/elections/README.md]: https://git.k8s.io/community/events/elections/README.md
+[voting exception form]: https://www.surveymonkey.com/r/k8s-sc-election-2018
+
+[the SQL query used by devstats for developer stats]: https://github.com/cncf/devstats/blob/master/metrics/shared/project_developer_stats.sql
```

Comment:
nit: this feels like a brittle way to do links. The link breaks if someone updates the sentence. The link slug style e.g. `[the SQL query used by devstats for developer stats][devstats-sql]` is a little bit more robust IMHO. But, just a nit.

Reply:
ack